test/support/cache_case.ex

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule EdgehogDeviceForwarder.CacheCase do
```

Comment:
This is unrelevant if this is the first PR that has to be merged, move this to the Cache PR